pkg/database/vercode.go

```diff
@@ -323,8 +316,8 @@ func (db *Database) ExpireCode(uuid string) (*VerificationCode, error) {
 
 // SaveVerificationCode created or updates a verification code in the database.
 // Max age represents the maximum age of the test date [optional] in the record.
-func (db *Database) SaveVerificationCode(vc *VerificationCode, maxAge time.Duration) error {
-	if err := vc.Validate(maxAge); err != nil {
+func (db *Database) SaveVerificationCode(vc *VerificationCode, realm *Realm) error {
```

Comment:
I'm wondering if `SaveVerificationCode` should dangle off the realm instead? A lot of its validation is tied to the realm, and it does have a FK relationship to the realm.